images/bootstrap/runner

```diff
@@ -68,6 +68,7 @@ fi
 
 # TODO(krzyzacy): remove after https://github.com/kubernetes/test-infra/pull/7267 is settled
 touch $(which kubetest)
+stat $(which kubetest)
```

Comment:
did we ever check that kubetest is in $PATH? this should be easy to debug locally

Reply:
e2e.go is able to find kubetest in PATH, https://github.com/kubernetes/kubernetes/blob/master/hack/e2e.go#L139 did not return error

Reply:
```
root@d3f966195e28:/workspace# echo $PATH
/go/bin:/usr/local/go/bin:/google-cloud-sdk/bin:/workspace:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

`/workspace` is part of PATH

Reply:
running locally via `docker run --rm -it --entrypoint=/bin/sh gcr.io/k8s-testimages/kubekins-e2e:latest`
```
# stat $(which kubetest)
  File: '/workspace/kubetest'
  Size: 8643782         Blocks: 16888      IO Block: 4096   regular file
Device: fe02h/65026d    Inode: 1198589     Links: 1
Access: (0750/-rwxr-x---)  Uid: (    0/    root)   Gid: (    0/    root)
Access: 2017-10-03 18:01:53.000000000 +0000
Modify: 2017-10-03 18:01:53.000000000 +0000
Change: 2018-03-22 19:03:16.723187898 +0000
 Birth: -
# touch $(which kubetest)
# stat $(which kubetest)
  File: '/workspace/kubetest'
  Size: 8643782         Blocks: 16888      IO Block: 4096   regular file
Device: fe02h/65026d    Inode: 2391425     Links: 1
Access: (0750/-rwxr-x---)  Uid: (    0/    root)   Gid: (    0/    root)
Access: 2018-03-22 19:03:44.674926636 +0000
Modify: 2018-03-22 19:03:44.674926636 +0000
Change: 2018-03-22 19:03:44.674926636 +0000
 Birth: -
```